### PR TITLE
add git trick

### PR DESCRIPTION
**Pull request by wikfeldt**

### CODE REVIEW

File: thor.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+useful Git trick to include line number stats:
+
+$ git diff --stat
```
Comment (wikfeldt) [inline thor.txt:3]: good trick!

**Review by wikfeldt — COMMENTED** [review]
yes thanks this is a good change

File: thor.txt (hunk 1 above)
Comment (johanhellsvik) [reply thor.txt:3]: Nice trick!

**Review by johanhellsvik — APPROVED** [review]
Nice trick!

**Pull request merged**